src/osd/scheduler/mClockScheduler.cc

```diff
@@ -76,15 +77,18 @@ void mClockScheduler::ClientRegistry::update_from_config(const ConfigProxy &conf
 const dmc::ClientInfo *mClockScheduler::ClientRegistry::get_external_client(
   const client_profile_id_t &client) const
 {
+  // rw_mtx must be held by the Caller
```

Comment:
Let's replace this comment with an assert that rw_mtx is held.

Reply:
The rw_mtx is now replaced with a ceph::mutex and the comment is replaced with assertion using ceph_mutex_is_locked() macro.